tests/test_doc.py

```diff
@@ -46,8 +47,8 @@
 ဆက်ခံထားသောမွန်အက္ခရာတို့မှ ဆင်းသက်လာသည်။
 """
 
-TEXT_7 = u"""\nHi <<First Name>>\nthis is filler text \xa325 more filler.\nadditilnal 
-filler.\nyet more\xa0still more\xa0filler.\n\xa0\nmore\nfiller.\x03\n\t\t\t\t\t\t    
```

Comment:
wan't the whitespace there on purpose?

Reply:
I don't think I changed anything here, at least not on purpose? Maybe my editor changed something... but which whitespace are you referring to?

Reply:
That's very likely. I was referring to  this whitespace:
![image](https://user-images.githubusercontent.com/586744/59347349-b3634300-8d14-11e9-8931-1a5bf814baf8.png)
Can you please remove this change from the PR?